src/components/nav/nav.js

```diff
@@ -3,20 +3,36 @@ import {Link} from 'react-router-dom'
 import { useCart } from '../../filter-context/cart-context'
 import { useAuth } from '../../filter-context/auth-context'
 import { useWishlist } from '../../filter-context/wishlist-context'
+import { useCategory } from '../../filter-context/category-context'
```

Comment:
create a single index file for import export of the files you need to write less import export statement that way

Reply:
ok got it